### DISCUSSION TITLE
Cannot connect to Typesense Self-hosted

### DISCUSSION THREAD

**Discussion by JohnDimopanagos** [bfritscher/typesense-dashboard#66] [General]
**Describe the bug**
I used the latest docker and run it on my ubuntu server where I host the self-hosted typesense server but the dashboard doesnt work, it saying not-found, even if I try your own and put my API and details it saying not found.

**Comment by bfritscher** [comment]
Hi, you need to provide more details on your setup, with versions and network topology. Are you sure you are connecting to the right ip/port and that its reachable if both are inside docker? if not on the same domain you need to enable cors on the server as mentionned in the readme.

**Comment by JohnDimopanagos** [comment]
Hi there, I followed the instructions and I have enable-cors = true on my typesense server. 

I have an ubuntu server 24.04.

I have installed the typesense using deb package, and it running fine on the server on port 8108.

I installed Nginx and then i created a host for mydomain.dev/api with SSL certificates using letsencrypt, and serving: proxy_pass to localhost:8108 . 

Then I installed the Dashboard using docker, on same server, port 8082:80 and added it to nginx on / and do proxy_pass to port: 8082, it opens the login page of dashboard, I put the domain, the ip, the private ip, the private ip of docker, and it doesnt work. 

it showing not found or network error.

**Comment by bfritscher** [comment]
I did not test with the server on a subfolder mapping, if possible create a proxy to a subdomain typesense.mydomain.dev
then you can specify this as domain, and use 443 as port since it will be behind the ssl proxy.
